ehcache-impl/src/test/java/org/ehcache/impl/internal/store/tiering/TieredStoreTest.java

```diff
@@ -560,6 +560,33 @@ public void CachingTierDoesNotSeeAnyOperationDuringClear() throws StoreAccessExc
       ArgumentMatchers.any(), ArgumentMatchers.any());
   }
 
+  @Test
+  public void AuthoritativeTierNullCheckDuringFlush() throws StoreAccessException, BrokenBarrierException, InterruptedException {
+    final TieredStore<String, String> tieredStore = new TieredStore<>(stringCachingTier, stringAuthoritativeTier);
+
+    final CyclicBarrier barrier = new CyclicBarrier(2);
+
+    doAnswer((Answer<Void>) invocation -> {
+      barrier.await();
+      barrier.await();
+      return null;
+    }).when(stringAuthoritativeTier).clear();
+    Thread t = new Thread(() -> {
+      try {
+        tieredStore.clear();
+      } catch (Exception e) {
+        throw new RuntimeException(e);
+      }
+    });
+
+    t.start();
+    barrier.await();
+    tieredStore.get("foo");
+    barrier.await();
+    t.join();
+    verify(stringAuthoritativeTier,never()).flush("foo",null);
```

Comment:
Can you fix the formatting here to confirm to some reasonable standard. A reasonable version could be:
```java
verify(stringAuthoritativeTier, never()).flush("foo", null);
```